### PR TITLE
updates conftest to reset coreir properly

### PR DESCRIPTION
**Pull request by rdaly525**

### CODE REVIEW

**Comment by rdaly525** [comment]
Travis is not letting this repo start builds. Looking into it

**Review by leonardt — APPROVED** [review]
LGTM

File: tests/rtl_utils.py [hunk 1]
```diff
@@ -48,9 +48,10 @@ def __hash__(self):
 # * https://github.com/StanfordAHA/lassen/issues/111
 # We reset the context because tests/test_micro.py calls compile and pollutes
 # the coreir context causing a "redefinition of module" error
-magma.backend.coreir_.CoreIRContextSingleton().reset_instance()
+#magma.backend.coreir_.CoreIRContextSingleton().reset_instance()
```
Comment (leonardt) [inline tests/rtl_utils.py:51]: Lets remove this comment

Comment (rdaly525) [reply tests/rtl_utils.py]: Fixed

**Pull request merged**